app/tests/unit/components/helpers/LoadingOnClickButton.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import LoadingOnClickButton from 'components/helpers/LoadingOnClickButton';
+import {mount} from 'enzyme';
+
+describe('the loading on click button', () => {
+  it('renders its children in its normal state', () => {
+    const rendered = mount(
+      <LoadingOnClickButton loadingText="loading">
+        test text
+      </LoadingOnClickButton>
+    );
+
+    expect(rendered).toMatchSnapshot();
```

Comment:
can you add assertions in those tests that check the button's content, so that we don't rely on reviewers checking the snapshots

Reply:
... and the `disabled` property too